releases/v1.1.0-rc.0.toml

```diff
@@ -0,0 +1,51 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.0.0"
+
+pre_release = true
+
+preface = """\
+`containerd` provides a daemon for managing running containers.
+
+1.1 is the second major release for `containerd` with added support for CRI, the
+Kubernetes Container Runtime Interface. CRI is a new default plugin which allows
+connecting the containerd daemon directly to a Kubernetes Kubelet to be used as
+the container runtime. The CRI GRPC interface listens on the same socket as the
+containerd GRPC interface and runs in the same process.
+
+In addition to all of the stability and bug fixes backported to 1.0,
+1.1 includes...
+
+- CRI plugin
+- ZFS snapshotter
```

Comment:
Do we want to include https://github.com/containerd/containerd/pull/2227, as well?

Reply:
Merged #2227 and updated